handler/health.go

```diff
@@ -20,6 +22,30 @@ func HandleHealth() http.HandlerFunc {
 			Version: version,
 			OK:      true,
 		}
-		WriteJSON(w, response, http.StatusOK)
+		status := http.StatusOK
+
+		if !checkInternetConnectivity() {
+			response = api.HealthResponse{
+				Version: version,
+				OK:      false,
+			}
+			status = http.StatusInternalServerError
```

Comment:
can we return a better error code here? And also, there should be some error message that we can log on the client side.

Reply:
sure will add a new field message in  HealthResponse